release notes/v0.39.0.md

```diff
@@ -1,71 +1 @@
-
-One of the upcoming next big steps is the introduction of a "time series" concept internally in k6 ([#2580](https://github.com/grafana/k6/issues/2580)). We'll start to efficiently group samples (i.e. metric measurements) with the same metric and tags into a single `TimeSeries`, which would unlock many of possibilities that were previously too inefficient to implement.
-
-Another upcoming metrics-related change would be the refactoring of [metric tags](https://k6.io/docs/using-k6/tags-and-groups/#tags) into two distinct types ([#2584](https://github.com/grafana/k6/issues/2584)) - ones that can be used for indexing (so, in the `TimeSeries` mentioned above, in [sub-metric thresholds](https://k6.io/docs/using-k6/thresholds/#thresholds-on-tags), in certain outputs that do aggregation based on tags, etc. ) and ones that cannot (e.g. high-cardinality metadata that's often unique for every data point).
-
-Unfortunately, we'll need to make a few minor breaking changes. The current [`url`, `error`, `vu` and `iter` system tags](https://k6.io/docs/using-k6/tags-and-groups/#system-tags) will be made non-indexable by default, to reflect their usually very high cardinality, so they won't be usable in thresholds. Instead of `error`, the `error_code` tag should be used there. And instead of `url`, users should already use the `name` tag to avoid issues, see the [documentation about URL grouping](https://k6.io/docs/using-k6/http-requests/#url-grouping). k6 v0.39.0 still supports thresholds with these tags, it will just print a warning if they are used.
-
-### Prometheus remote-write output
-
-Because of these changes, we'll finally be able to refactor and improve the [Prometheus remote-write output extension](https://github.com/grafana/xk6-output-prometheus-remote/). Our goal is to make it more efficient, polish its rough edges, and merge it into the k6 core as a built-in output module in the near future.
-
-### Tracing support
-
-Another new feature these changes would unlock is support for distributed tracing ([#2128](https://github.com/grafana/k6/issues/2128)).
-
-Because of the upcoming support for high-cardinality unindexable metric tags, we'll be able to safely attach things like the unique trace and span IDs to every metric measurement without negatively affecting other parts of k6.
\ No newline at end of file
+k6 v0.39.0 was never released because of problems with our CI release process. Howeever, because the v0.39.0 git tag already exists, we have left this file here. Please see the k6 v0.39.1 release notes instead for the actual release details.
```

Comment:
```suggestion
k6 v0.39.0 was never released because of problems with our CI release process. However, because the v0.39.0 git tag already exists, we have left this file here. Please see the k6 [v0.39.1](https://github.com/grafana/k6/blob/master/release%20notes/v0.39.1.md) release notes instead for the actual release details.
```

Reply:
:-1: this link won't work in the [releases page](https://github.com/grafana/k6/releases), and I don't think we can make a link that will work in both places

Reply:
```suggestion
k6 v0.39.0 was never released because of problems with our CI release process. However, because the v0.39.0 git tag already exists, we have left this file here. Please see the k6 v0.39.1 release notes instead for the actual release details.
```
Then we just need to fix this typo.